### DISCUSSION TITLE
Final Presentation

### DISCUSSION THREAD

**Discussion by LilacHo** [UndefiedCse/Heat-Island#58] [General]
Powerpoint: https://docs.google.com/presentation/d/1pA4czEW8QuC-m--yKppEvx0GkmPazlYCRd-mOsqeVpY/edit?usp=sharing
A good powerpoint example: https://github.com/sliwhu/UWHousingTeam/blob/master/doc/Data515-Final%20presentation.pdf

- Background. Describe the problem or area being addressed. @JornsenChao 
- Data used. What data did you use? How was it obtained? What are its limitations? @JornsenChao @LilacHo 
- Use cases. How users will interact with your system in a way that addresses the problem area. @LilacHo @loriwon-uw 
- Demo. Demonstrate your software. @loriwon-uw 
- Design. Describe the components and how they interact to accomplish the use cases. @loriwon-uw (I think I signed up for it as well? @LilacHo )
- Project Structure. Show the structure of your github repository. We can do it together at the end.
- Lessons learned and future work. Focus on software engineering lessons. @krittin-kulrat

**Comment by LilacHo** [comment]
Lessons learned and future work ideas

**Comment by JornsenChao** [comment]
Hi Lilac,
please give edit permission, thanks
@LilacHo

**Reply by loriwon-uw** [reply]
Are you logged in to your uw account? If so, you should have editing access.  At least I think I see you on the list of editable people?

**Reply by JornsenChao** [reply]
Thanks, I was using my github email. Thanks for reminding!
